<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formcadastrodisciplina</class>
 <widget class="QDialog" name="formcadastrodisciplina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>301</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Professor da Disciplina (CPF)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="professor_disciplina"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Código</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cadastro_codigo"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cadastro_nome"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Carga Horária</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cadastro_cargahoraria"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salvar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
